### PR TITLE
New keyboard shortcuts for managing windows and tabs

### PR DESCRIPTION
**Pull request by silvasch**
Implements #961

### CODE REVIEW

File: crates/rnote-ui/src/appwindow/actions.rs [hunk 1]
```diff
@@ -567,13 +565,6 @@ impl RnAppWindow {
             appwindow.handle_widget_flags(widget_flags, &canvas);
         }));
 
-        // New doc
```
Comment (flxzt) [inline crates/rnote-ui/src/appwindow/actions.rs:570]: This action is still needed for the `New` menu item in the app-menu.

File: crates/rnote-ui/src/app/mod.rs [hunk 2]
```diff
@@ -128,7 +128,7 @@ mod imp {
         }
 
         /// Initializes and shows a new app window
-        fn new_appwindow_init_show(&self, input_file: Option<gio::File>) {
+        pub fn new_appwindow_init_show(&self, input_file: Option<gio::File>) {
```
Comment (flxzt) [inline crates/rnote-ui/src/app/mod.rs:131]: use `pub(crate)` here

File: crates/rnote-ui/src/appwindow/actions.rs (hunk 1 above)
Comment (silvasch) [reply crates/rnote-ui/src/appwindow/actions.rs:570]: True. But I don't think it should use the old `new-doc` action, but should also create a new tab. I don't think replacing your current tab is the desired behaviour.

Comment (flxzt) [reply crates/rnote-ui/src/appwindow/actions.rs:570]: Let's keep the action anyway, in case it is needed again at some point in the future.

**Comment by flxzt** [comment]
looks good now. thanks

**Pull request merged**